packages/worker/native/src/lib.rs

```diff
@@ -146,10 +146,14 @@ fn worker_complete_task(mut cx: FunctionContext) -> JsResult<JsUndefined> {
 }
 
 fn worker_shutdown(mut cx: FunctionContext) -> JsResult<JsUndefined> {
```

Comment:
Doesn't need to be this PR but you'll probably want to have some mechanism to drain any remaining activations from `poll_task` until it returns `CoreError::ShuttingDown`. Remaining activations may exist due to replaying a workflow to catch up to the end of history which was already fetched from the server

Reply:
That's a really good point (see #19).